tests/test_domain_py.py

```diff
@@ -44,3 +46,44 @@ def test_function_signatures():
 
     rv = parse('func(a=[][, b=None])')
     assert text_type(rv) == u'a=[], [b=None]'
+
+@with_app('html', testroot='domain-py')
+def test_memory_address(app, status, warning):
```

Comment:
I don't think `test_domain_py.py` is the appropriate place for this test (it really has nothing to do with Python domain). I think you should better place it in a separate file.